src/Uno.UWP/Helpers/PlatformImageHelpers.skia.cs

```diff
@@ -14,7 +14,8 @@ internal static partial class PlatformImageHelpers
 
 	internal static Task<string> GetScaledPath(Uri uri, ResolutionScale? scaleOverride)
 	{
-		var path = uri.PathAndQuery;
+		// Unescaping is necessary for things like spaces that are valid in a file system path but not in a URI
+		var path = Uri.UnescapeDataString(uri.AbsolutePath);
```

Comment:
So, no need for the query here?  (PathAndQuery vs AbsolutePath)

Reply:
This is specifically for local paths, so the query should always be empty.